### DISCUSSION TITLE
Generator Types

### DISCUSSION THREAD

**Discussion by OneLemonyBoi** [OneLemonyBoi/MiniUtilities#81] [Ideas]
Put generator types that you want added here!

**Comment by DeadSlayer54** [comment]
Seeing the curseforge page, I see you already have solar panels in your description. Ideas for power generators:
-Multi-Tiered Solar Panels (progression system for more generation)
-The generators on the XU/XU2 (Especially the Nether Star generator)
-Not exactly a generator but maybe a power translator block? Convert from
  -EU (Industrialcraft)
  -gJ (Galacticraft)
  -J (Mekanism)
  -MJ (BuildCraft)
  -RF (Thermal Expansion)
  -W (RotaryCraft)
  (20RF = 2MJ = 5EU = 50J)
  (source: https://ftbwiki.org/Power_Systems#List_of_Energy_Types)
  A tall order haven't even used rotary nor buildcraft more often using RF or ME, this was more on the idea of AE2's Power Accepter but with more versatility in mind for more cross-mod helpers.
  
(these are more ideas than requests but felt they could be kinda fun)
-Water Wheel: Idea from Immersive Engineering, generate power by the flow of water
-Dimensional Generator: Weak generator powered by a nearby nether portal, but nether mobs are hostile to it so any that pass through will attack it. Near an end portal is more powerful but mobs will attack that too.
-Lightning Generator: Doesn't actually produce electricity but rather when it rains it has a chance to be struck which will store a ton of energy in it to be used elsewhere (could be cool to build a trap on top to hold mobs like horses or creepers for lightning-spawned mobs)
-Heat Generator: Passive generator that's active when above a fire or lava source
-Potential Energy Generator: The higher the block is above sea level the more energy it produces
-Falling Generator: The farther the generator falls the more energy is produced
-Blast Generator: Generates power by explosions happening around it, like creepers, TNT, or other mods' explosive addons, like that flower from botania
